cmd/fluxctl/root_cmd.go

```diff
@@ -49,7 +54,10 @@ func (opts *rootOpts) Command() *cobra.Command {
 		SilenceErrors:     true,
 		PersistentPreRunE: opts.PersistentPreRunE,
 	}
-	cmd.PersistentFlags().StringVarP(&opts.URL, "url", "u", "https://cloud.weave.works/api/flux",
+
+	cmd.PersistentFlags().StringVarP(&opts.Namespace, "k8s-forward-namespace", "f", "default",
```

Comment:
Not sure how I feel about having a shorthand for this -- and `-f` is usually used for `file` (albeit we don't have any commands that accept a file; and, we are stuck with `-u` and `-t` for connection flags, which I wouldn't have chosen ..).

We can shorten the long name, it is a _little_ long: `--k8s-fwd-namespace` or even `--k8s-fwd-ns`. wdyt?

Reply:
Makes sense to me, changed.